sdk/endpoint/mutators.go

```diff
@@ -3,8 +3,12 @@ package endpoint
 import (
 	"context"
 
+	"github.com/sirupsen/logrus"
+
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/local/connection"
+	"github.com/networkservicemesh/networkservicemesh/k8s/pkg/networkservice/namespace"
```

Comment:
Ah... this is why the k8s dependencies suddenly popped up everywhere... please *don't* put dependencies from k8s/ into other places like sdk/ controlplane/ etc...  we are trying to keep them minimal.  

It looks like the [namespace stuff](https://github.com/networkservicemesh/networkservicemesh/blob/master/k8s/pkg/networkservice/namespace/nsm.go) should migrate to someplace like [pkg/tools](https://github.com/networkservicemesh/networkservicemesh/tree/master/pkg/tools) to break this dependency.